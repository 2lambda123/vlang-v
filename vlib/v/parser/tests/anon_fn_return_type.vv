struct Msg {
	name string
	text string
}
ms := &Msg{}

fn (m &Msg) one() { 
}
