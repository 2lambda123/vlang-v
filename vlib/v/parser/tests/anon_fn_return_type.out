vlib/v/parser/tests/anon_fn_return_type.vv:7:13: error: invalid return type `one` for anonymous function
    5 | ms := &Msg{}
    6 | 
    7 | fn (m &Msg) one() { 
      |             ~~~
    8 | }
